bundles/org.eclipse.osgi.tests/src/org/eclipse/osgi/tests/bundles/ClassLoadingBundleTests.java

```diff
@@ -2149,7 +2158,8 @@ public void testDefaultLocalUninstall() throws Exception {
 
 	@Test
 	public void testBug490902() throws BundleException, InterruptedException, ClassNotFoundException,
-			InstantiationException, IllegalAccessException {
+			InstantiationException, IllegalAccessException, IllegalArgumentException, InvocationTargetException,
+			NoSuchMethodException, SecurityException {
```

Comment:
For test cases that throw that many different type of exception, I would just specify `throws Exception`.
If an exception is throw its always an error.